cypress-tests/cypress/e2e/configs/Payment/Braintree.js

```diff
@@ -143,6 +143,9 @@ export const connectorDetails = {
       },
     },
     "3DSAutoCapture": {
+      Configs: {
+        TRIGGER_SKIP: true,
+      },
```

Comment:
why skipping? unless it is really not possible to achieve the redirection this should be used. i would suggest you modify the redirectionHandler instead and add support for braintree 3ds instead.